spec/scenic/schema_dumper_spec.rb

```diff
@@ -20,4 +20,15 @@ class Search < ActiveRecord::Base; end
 
     expect(Search.first.haystack).to eq "needle"
   end
+
+  it "does not dump views belonging to Posgresql extensions" do
+    Search.connection.enable_extension("pg_stat_statements")
+    Search.connection.reconnect!
+    stream = StringIO.new
+
+    ActiveRecord::SchemaDumper.dump(Search.connection, stream)
+
+    output = stream.string
+    expect(output).not_to include "create_view :pg_stat_statements"
```

Comment:
I can't get this test to fail on my machine when I don't have the ignore_tables additions, so I think its more environment dependent than we would like.